debian/control

```diff
@@ -5,6 +5,7 @@ Maintainer: Matthias Urlichs <matthias@urlichs.de>
 Build-Depends: debhelper (>= 10),
   autoconf, automake, libtool,
   git,
+  libev4,
```

Comment:
Sorry but that can't be correct. libev-dev is already in the list and that already depends on libev4.